app/assets/stylesheets/application.scss

```diff
@@ -76,3 +75,17 @@
     margin: 0;
   }
 }
+
+#map-container {
+   height: 400px;
+   border-radius: 16px 16px;
+   border-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces